gradle.properties

```diff
@@ -0,0 +1,16 @@
+GROUP=com.pinterest.ktlint
+VERSION_NAME=0.3.4-SNAPSHOT
```

Comment:
Previously, master `SNAPSHOT` builds were always VERSION_NAME `0.0.0-SNAPSHOT`. Whenever a new version was released, the `VERSION_NAME` was not committed, just changed locally on the deploying machine.

What's more typical for OSS projects? I'm fine with committing the version changes, or always keeping it at 0.0.0.

Reply:
Ideally you want a versioned snapshot, this way ktlint users can test new features on the fly when the feature is available. They can then explain which version has problems, etc.

For instance Okio, Okhttp and Retrofit use it, see here: https://github.com/square/okio/blob/f790abfeabb4b6dbeeb49147561404f58ffb53cf/gradle.properties#L2.

Here is the maven snapshot repo for Okio: https://oss.sonatype.org/content/repositories/snapshots/com/squareup/okio/okio/. You can see the versioned uploads.

Reply:
Sounds good to me. Can you update `VERSION_NAME` to `0.34.0-SNAPSHOT` then?

Reply:
Oh, I didn't realize I had a typo here. This should be `0.34.0-SNAPSHOT`.